src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/PEReaderExtensions.cs

```diff
@@ -135,5 +138,21 @@ public static bool TryGetReadyToRunHeader(this PEReader reader, out int rva)
         {
             return reader.GetExportTable().TryGetValue("RTR_HEADER", out rva);
         }
+
+        internal static bool IsReadyToRunPlatformNeutralSource(this PEReader peReader)
```

Comment:
```suggestion
        /// <summary>
        /// Check whether the file is a ReadyToRun image created from platform neutral (AnyCPU) IL image.
        /// </summary>
        /// <param name="reader">PEReader representing the executable to check</param>
        /// <returns>true when the PEReader represents a ReadyToRun image created from AnyCPU IL image, false otherwise</returns>
        public static bool IsReadyToRunPlatformNeutralSource(this PEReader peReader)
```
All methods in this file have comment headers.

Reply:
I added it as an internal API since it's compiled as part of this project and not needed in other place (like crossgen2 which requires public)

Reply:
I added it as an internal API since it's compiled as part of this project and not needed in other place (like crossgen2 which requires `public static bool TryGetReadyToRunHeader(this PEReader reader, out int rva)`). We don't usually add XML docs on internal APIs.